docs/developer/contributors.md

```diff
@@ -0,0 +1,139 @@
+# Contributor guidance
+
+## Issues
+
+Well crafted issues improve the speed that work can be triaged, understood, and completed. To help guide this process [issue templates](https://github.com/Microsoft/fast-dna/tree/master/.github/ISSUE_TEMPLATE) exists to allow users to choose an appropriate bug, issue, or custom template.
+
+Here's a good resource for [Do's and Don'ts](https://hackernoon.com/45-github-issues-dos-and-donts-dfec9ab4b612) when working with bugs, issues, and requests.
+
+[GitHub Labels](https://developer.github.com/v3/issues/labels/) are used to manage issues broadly considered work items grouped by color and prefixed by label name.
+
+## Pull requests
+
+When landing a pull request ("PR"), individuals must modify the original commit message to include comprehensive details. This detail along with other git history meta information deploys as part of the documentation changelog history. Changlogs are generated by [Lerna](https://lernajs.io/) using [Conventional commits](https://conventionalcommits.org/).
+
+PR comments should include the issue number per the [github closing issue keywords](https://help.github.com/articles/closing-issues-using-keywords/) documentation.
+
+Each PR, analyzes changing code to automatically assign [Code Owners](https://github.com/Microsoft/fast-dna/blob/master/.github/CODEOWNERS) based on the different configurations for packages and file types.
+
+### Formatting title
```

Comment:
Title removed.  Wasn't needed after all.  Sounds like otherwise no action taken. We all agree we don't want to teach people how to do things they should already know. ie: git, github, etc.